source/agora/node/TransactionPool.d

```diff
@@ -98,7 +100,8 @@ public class TransactionPool
 
     ***************************************************************************/
 
-    public this (ManagedDatabase db)
+    public this (ManagedDatabase db,
+            DoubleSpentSelector double_spent_selector = toDelegate(&TransactionPool.defaultSelector))
```

Comment:
Don't think we want the default value, since it's only instantiated a handful of times.
For unittests, there's a specific ctor.